docs/src/pages/careers/react-engineer-x.md

```diff
@@ -1,6 +1,6 @@
-# React Engineer - X (future)
+# React Engineer - X
```

Comment:
We should update the /careers page to be in sync with this change.

Reply:
I've added the `(future)` back since we are not actively looking right now.